Offseason2022/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -332,7 +332,7 @@ private void configureButtonBindings( )
     // final AxisTrigger operRightTrigger = new AxisTrigger(m_operator, XboxController.Axis.rightY);
 
     // Operator - A, B, X, Y
-    operA.whenPressed(new IntakeDeploy(m_intake, false), true);
+    operA.whenPressed(new IntakeDeploy(m_intake, true), true);

```

Comment:
I think this is supposed to be false and was correct. It is for retracting the intake manually for special cases. I  know it's a little non-obvious, but that was the purpose.